src/platform/site-wide/sass/shame.scss

```diff
@@ -571,3 +571,27 @@ img[data-srcset] {
   border-top-left-radius: 0;
   border-bottom-left-radius: 0;
 }
+
+.no-p-bottom-margin p {
+  margin-bottom: 0;
```

Comment:
we should use the design system utilities to avoid adding duplicate css: 

https://design.va.gov/utilities/margins#margin-bottom

Reply:
Unfortunately, this can't be done for the element that we are targeting, without impacting other elements that we don't want to apply this rule. In this instance, we need to target p tags that are input by users in a drupal wysiwyg. If I were to apply this rule to vads-u-margin-bottom--0 p tags, there would be many instances of p tags not entered in the drupal wysiwyg for the operating status page that would be unintentionally effected, @rianfowler